server/src/models/diets.js

```diff
@@ -1,4 +1,7 @@
 // @flow
+import type { Category } from "models/category";
```

Comment:
Usually we put type imports below regular imports

Reply:
Will do next time!